AI Toolkit docs migration

sffamily opened the pull request.

ntrogh commented on docs/intelligentapps/faq.md:
```diff
@@ -0,0 +1,84 @@
+
+# AI Toolkit FAQ
+
+### There are too many fine-tune settings do I need to worry about all of them?
```
Can you group these FAQs into relevant sections? This a long list and not very scannable for readers.

ntrogh replied:
The heading level seems off, unless you want to add an H2 to group of course.

sffamily replied:
revised

ntrogh commented on docs/intelligentapps/faq.md:
```diff
@@ -0,0 +1,84 @@
+
+No, you can just run with the default settings and our current dataset in the project to test. If you want you can also pick your own dataset but you will need to tweak some setting see [this](walkthrough-hf-dataset.md) tutorial for more info.
+
+### AI Toolkit Preview would not scaffold the fine-tuning project
```
Preview?

ntrogh commented on docs/intelligentapps/finetune.md:
```diff
@@ -0,0 +1,278 @@
+
+# Finetune models
+
+# **[Preview]** Introducing Remote Development with AI Toolkit for VS Code
```
Don't use H1 except for the title.

Why is remote development part of an article about finetuning?

sffamily replied:
a little confusing, so had this rewritten.

ntrogh commented on docs/intelligentapps/finetune.md:
```diff
@@ -0,0 +1,278 @@
+---
+
+# Finetune models
+
```
Add an introduction about what finetuning is and what this article is about. This article is very long and we can't expect readers to have to go through the entire article to know if this is relevant for them.

ntrogh commented on docs/intelligentapps/finetune.md:
```diff
@@ -0,0 +1,278 @@
+In addition to local development, the AI Toolkit for VS Code now supports remote development. This feature enables users to provision for Azure Container Apps to run model fine-tuning and inference endpoint in the cloud.
+
+## Prerequisites
+1. To run the model fine-tuning and inference in your remote Azure Container Apps Environment, make sure your subscription has enough GPU capacity. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the required capacity for your application. [Get More Info about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
```
These are a lot of steps to put under a prerequisites section. Can these steps be a "Set up your project/environment" section?

sffamily replied:
done

ntrogh commented on docs/intelligentapps/finetune.md:
```diff
@@ -0,0 +1,278 @@
+
+> **Note:** The project currently works either locally or remotely within the AI Toolkit for VS Code. If you choose *"Fine-tune locally"* during project creation, it will operate exclusively in WSL without remote development capabilities. On the other hand, if you forego enabling *"Fine-tune locally"*, the project will be restricted to the remote Azure Container App environment.
+
+## What's Next
```
Don't use a what's next heading in technical docs, especially weird in the middle of an article.

sffamily replied:
removed this section

ntrogh commented on docs/intelligentapps/finetune.md:
```diff
@@ -0,0 +1,278 @@
+Refer to [Fine-Tuning models remotely](#fine-tuning-models-remotely) for instructions on how to fine-tune a model in Azure Container Apps.
+To learn how to create an inference API endpoint in Azure Container Apps, see I[Inferencing with the fine-tuned model](#inferencing-with-the-fine-tuned-model).
+
+# Fine-Tuning models remotely
```
Heading levels

sffamily replied:
fixed

ntrogh commented on docs/intelligentapps/finetune.md:
```diff
@@ -0,0 +1,278 @@
+ContentId:
+PageTitle: Finetune AI Models
+DateApproved:
+MetaDescription: Use custom dataset to finetune a generative AI model in the Azure cloud or locally with GPUs. Deploy the finetuned model to the Azure cloud or download adapter file.
```
What is an "adapter file"? This text shows up in Google/Bing - people will not know what this concept is about. Make sure to keep this text as clear as possible so people know it's relevant just from search results.

sffamily replied:
updated

sffamily commented on docs/intelligentapps/get_started.md:
Merged into overview as "Quick start" section.

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,36 @@
+Search the Visual Studio Code extension marketplace for "AI Toolkit", install either the formal released version to stable features, or pre-released version for early access of new features. Check What's New during the installation for detailed feature list of each version.
+![alt text](./images/overview/install.png)
+
+## Key features
```
added to intro paragraph.

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,36 @@
+MetaSocialImage:
+---
+
+# Overview of AI Toolkit
```
updated.

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,36 @@
+- Attachment support for Multi-modal Language Models
+- Many new features coming soon: evaluation, dataset, batch run, finetune, prompt...
+
+Follow [Get started guide](./get_started.md) or find docs on [models](./models.md) and [playground](./playground.md)
```
updated

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,36 @@
+
+## Key features
+
+- Get started guide
```
updated

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,36 @@
+
+## Install and setup
+
+Search the Visual Studio Code extension marketplace for "AI Toolkit", install either the formal released version to stable features, or pre-released version for early access of new features. Check What's New during the installation for detailed feature list of each version.
```
this is great. added.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+## Models supported
+
+AI Toolkit support a boradrange of generative AI models. Both Small Language Models (SLM) and Large Language Models (LLM) are supported.
```
added into the top intro.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+# Models in AI Toolkit
+
+
+## Models supported
```
updated

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+## Find a model
+
+On AI Toolkit, click the "Models" on treeview to open the model catalog, as shown in step 1.
```
done.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+On AI Toolkit, click the "Models" on treeview to open the model catalog, as shown in step 1.
+![alt text](./images/models/model_catalog_new.png)
+
+There are a set of filters on model catalog as shown in step2:
```
updated.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+    Depends on the local availability, to filter models by CPU, GPU and NPU.
+
+Additional filters to filter modes that can run locally or remotely, with default to include all model types.
```
removed.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+## License and sign-in
+
+Some of the models require publisher or hosting service license and account to sign-in.
```
added.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+Additional filters to filter modes that can run locally or remotely, with default to include all model types.
+
+Fine-tuning Support switch filters only the model that can be used to run finetuning.
```
made it clear.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+## Find model details
+
+Click on each model card in model catalog to find more details on the model.
```
thanks for raising this, updated.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+## Select a model for test
+
+On each model card, there are several options:
```
added.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+## Bring Your Own Models
+
+AI Toolkit's playground supports remote models. Mouse-over "MY MODELS" on treeview, a "+" sign appears to add a remote model into AI Toolkit. Fill in the requested information on model name, display name, model hosting URL and optional auth string. A remote model is added and shown in treeview as step3.
```
updated

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+![alt text](./images/models/byom.png)
+
+## Ollama models
+
```
added intro.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,92 @@
+
+    > Please note that it will only show models that are already downloaded in Ollama and not already added to AI Toolkit. In order to download a model from Ollama you can run `ollama pull <model-name>`. You can see the list of models supported by Ollama in [Ollama library](https://ollama.com/library) or refer to [Ollama documentation](https://github.com/ollama/ollama).
+
+1. You will see it on treeview's **MY MODELS** list. Use it the same way as other models in playground.
```
added

sffamily commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,106 @@
+
+Playground is where developer to interact with AI models of choice, try different prompts with different model parameter settings. For the most recent multi-modal models that supports attachment of different format, developer can also interact through playground.
+
+In AI Toolkit, click the playground or from Model Catalog to load or try in playground, the playground view will display as screenshot:
```
rewritten.

sffamily commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,106 @@
+
+![alt text](./images/playground/playground.png)
+
+User must select a model from the choice to start with. Optionally user can insert context instructions or change model parameters. User prompts or questions are entered from the chat box at the bottom of playground, with options to send, clear chat history, or add attachement for the prompt.
```
fixed

sffamily commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,106 @@
+
+# Add attachment for Multi-modal models
+
+If the model doesn't support attachment yet, the icon will be disabled. Otherwise click the pin icon, follow the instruction to attach one or more local files and use them together in prompt.
```
added

sffamily commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,106 @@
+- AI Toolkit v0.4.0 or newer.
+- OpenAI compatible chat completion endpoint.
+
+## How to add a remote model
```
Good point, they are the same. So kept the part in models doc, and removed this as duplicates.

sffamily commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,106 @@
+
+    > Note: AI Toolkit will send the chat history, context instructions and parameters to the endpoint, but it depends on the actual endpoint and model whether they are supported.
+
+## FAQ
```
moved this out into faq doc.

sffamily commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,106 @@
+
+Users can chat with remotely hosted language model in playground.
+
+## Prerequisites
```
deleted this part.

sffamily commented:
@ntrogh I just went through and addressed all your 1st pass comments. In addition I added 2 new pages for features we are releasing this week. Also did some minor updates of contents. Please help review.

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+Once you find the model of interest, you can test run this model for chat completeions in the playground.
+
+On each model card, there are several options:
+- **Try in Playground** link that can load selected model in playground for test without model downloading.
```
```suggestion
- **Try in Playground**: load the selected model for testing in the playground without downloading it
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+
+## Find a model
+
+On AI Toolkit, select **Models** in the treeview to open the model catalog, as shown in step 1.
```
```suggestion
To find a model in the model catalog:

1. In the AI Toolkit view, select **CATALOG** > **Models* to open the model catalog
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+
+On each model card, there are several options:
+- **Try in Playground** link that can load selected model in playground for test without model downloading.
+- **Download** link that will download the model to local first from source like Hugging Face.
```
```suggestion
- **Download**: download the model from a source like Hugging Face
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+On each model card, there are several options:
+- **Try in Playground** link that can load selected model in playground for test without model downloading.
+- **Download** link that will download the model to local first from source like Hugging Face.
+- **Load in Playground** will load the downloaded model into playground for chat.
```
```suggestion
- **Load in Playground**: load a downloaded model into the playground for chat
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+
+AI Toolkit's playground also supports remote models.
+
+If you have a self hosted or deployed model that is accessible from internet, you can add it to AI Toolkit and use it in the playground.
```
```suggestion
If you have a self-hosted or deployed model that is accessible from the internet, you can add it to AI Toolkit and use it in the playground.
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+
+1. Select **Add an Ollama model**
+
+    ![Select type](./images/models/select-type.png)
```
Improve the alt text to be more descriptive - do this for all alt texts please

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+
+    ![Select type](./images/models/select-type.png)
+
+1. Select **Select models from Ollama library**. Or if you start Ollama runtime at a different address, you can choose "Provide custom Ollama endpoint" to specify an Ollama endpoint.
```
```suggestion
1. Select **Select models from Ollama library**. Or if you start Ollama runtime at a different address, you can choose **Provide custom Ollama endpoint** to specify an Ollama endpoint.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+# AI Toolkit playground
+
+The AI Toolkit playground is where you can interact with your AI models and try different prompts with different model parameter settings. You can also use the playground to interact with multi-modal models that support attachment of different input formats.
+
```
```suggestion

## Test a model in the playground

```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+
+The AI Toolkit playground is where you can interact with your AI models and try different prompts with different model parameter settings. You can also use the playground to interact with multi-modal models that support attachment of different input formats.
+
+In AI Toolkit, select the `Playground` in the treeview, or select `Load in Playground`/`Try in Playground` from a model card in the model catalog, you will enter playground view as screenshot:
```
```suggestion

There are multiple options to open the model playground:

- In AI Toolkit, select the **Playground** in the treeview

- Select **Load in Playground** or **Try in Playground** from a model card in the model catalog
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+
+In AI Toolkit, select the `Playground` in the treeview, or select `Load in Playground`/`Try in Playground` from a model card in the model catalog, you will enter playground view as screenshot:
+
+![Playground view](./images/playground/playground.png)
```
Move this screenshot to the intro because it just provides an overview of the playground.

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+
+![Playground view](./images/playground/playground.png)
+
+To get started, perform the steps:
```
```suggestion
To test a model in the playground:
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+1. Type prompts or questions from the chat box at the bottom of playground
+1. Send questions and wait for responses.
+
+From the chat box, you can also clear chat history, or add attachement for the prompt.
```
```suggestion
From the chat input box, you can also clear chat history or add attachments for the prompt.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+1. Change model parameters (Optional)
+1. Type prompts or questions from the chat box at the bottom of playground
+1. Send questions and wait for responses.
+
```
@Add screenshots and provide info about the parameters or instructions.

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+
+## Add attachments for multi-modal models
+
+Multi-modal models allow you to include attachment files in different format, such as image, voice, video, or document, send together with prompts to AI models. You can ask questions about the contents in the attachements.
```
```suggestion
Multi-modal models allow you to include attachment files in different format, such as image, voice, video, or document, send together with prompts to AI models. You can ask questions about the contents in the attachments.
```

ntrogh commented on docs/intelligentapps/playground.md:
```diff
@@ -0,0 +1,38 @@
+
+Multi-modal models allow you to include attachment files in different format, such as image, voice, video, or document, send together with prompts to AI models. You can ask questions about the contents in the attachements.
+
+For the models that support attachments, the pin icon will show. Select the pin icon, follow the instruction to attach one or more local files and use them with prompt.
```
```suggestion
For the models that support attachments, the attachment icon (paperclip) will show. Select the icon, and follow the instructions to attach one or more local files and use them with your prompt.
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -0,0 +1,42 @@
+MetaSocialImage:
+---
+
+# Bulk Run Overview
```
Instead of using <feature name> overview, describe what it does: "Run multiple prompts in bulk"

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -0,0 +1,42 @@
+
+# Bulk Run Overview
+
+Bulk run feature in AI Toolkit allows you to run many prompts in batch mode. Compared to playground you can only run each prompt manually in order.
```
```suggestion
The bulk run feature in AI Toolkit allows you to run multiple prompts in batch. When you use the playground, you can only run one prompt manually at a time, in the order they're listed.
```

ntrogh replied:
Add more information about how this works because in the next section you're mentioning a dataset. Explain briefly how the batch thing works.

sffamily replied:
updated.

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -0,0 +1,42 @@
+
+## Start a bulk run
+
+Navigate to `Bulk Run` in the `TOOLS` section in the tree view of AI Toolkit. You can select either a sample dataset to get familiar about the feature, or import a local JSONL file that has at least a field `query` to use as prompts.
```
Use numbered steps to make it easier for users to know what to do. Also include screenshots where needed.

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -0,0 +1,42 @@
+Navigate to `Bulk Run` in the `TOOLS` section in the tree view of AI Toolkit. You can select either a sample dataset to get familiar about the feature, or import a local JSONL file that has at least a field `query` to use as prompts.
+
+
+## Run any prompt
```
This heading is not very clear

sffamily replied:
updated to "Run a single prompt"

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -0,0 +1,36 @@
+
+# Evaluation Overview
+
+AI engineers often need to evaluate models with different parameters, different prompts in a dataset for comparing to ground truth and generate values from the comparisons. AI Toolkit allows you do perform evaluations with minimal efforts.
```
```suggestion
AI engineers often need to evaluate models with different parameters, different prompts in a dataset for comparing to ground truth and generate values from the comparisons. AI Toolkit allows you to perform evaluations with minimal effort.
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -0,0 +1,36 @@
+
+## Prepare an evaluation job
+
+Navigate to the tree view's `Batch Run` entry in `TOOLS` section. You will see an evaluation job history view. Select the button `Start Evaluation`, then answer the questions:
```
Use numbered steps to provide instructions, don't just add a sentence because it's not very easy to parse.
Include a screenshot where needed

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -0,0 +1,36 @@
+
+AI engineers often need to evaluate models with different parameters, different prompts in a dataset for comparing to ground truth and generate values from the comparisons. AI Toolkit allows you do perform evaluations with minimal efforts.
+
+## Prepare an evaluation job
```
What does it mean to prepare an evaluation job? Is this just a technical step before you can run, or is something actually being done? If nothing special, include it as part of starting an evaluation job.
At least describe what this is.

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -0,0 +1,36 @@
+
+## Start an evaluation job
+
+You will be promoted to confirm you want to start the evaluation. Or select any individual evaluation job to re-run it.
```
```suggestion
You will be prompted to confirm that you want to start the evaluation. Alternatively, select any individual evaluation job to re-run it.
```

ntrogh replied:
This section is very short - combine with previous section?

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -0,0 +1,36 @@
+
+## Monitor the evaluation job
+
+Once an evaluation job is started, you can find its status from the evaluation job overview. Each evaluation job has link to the dataset used, logs from evaluation process, timestamp and details of the evaluation.
```
Screenshot?

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -0,0 +1,36 @@
+
+## Find results of evaluation
+
+Select the evaluation job detail, the view has columns of selected evaluators with the numerical values. Some may have aggregate values.
```
Screenshot? This one sentence is not providing a lot of value.
Also include a link to the Data Wrangler extension and docs.

ntrogh reviewed (changes requested):
@sffamily more feedback

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,48 @@
+Search the Visual Studio Code extension marketplace for "AI Toolkit" and <a class="install-extension-btn" href="vscode:extension/ms-windows-ai-studio.windows-ai-studio">install the AI Toolkit for VS Code</a>
+
+You can switch the installation between the formal released version for stable features and pre-released version for early access of new features. Check What's New during the installation for detailed feature list of each version.
+![AI Toolkit in the marketplace](./images/overview/install.png)
```
agreed, removed.

sffamily commented on docs/intelligentapps/models.md:
```diff
@@ -0,0 +1,98 @@
+## Find a model
+
+On AI Toolkit, select **Models** in the treeview to open the model catalog, as shown in step 1.
+![Select model in model catalog](./images/models/model_catalog.png)
```
replaced screenshot.

sffamily commented on docs/intelligentapps/overview.md:
```diff
@@ -0,0 +1,48 @@
+
+Search the Visual Studio Code extension marketplace for "AI Toolkit" and <a class="install-extension-btn" href="vscode:extension/ms-windows-ai-studio.windows-ai-studio">install the AI Toolkit for VS Code</a>
+
+You can switch the installation between the formal released version for stable features and pre-released version for early access of new features. Check What's New during the installation for detailed feature list of each version.
```
might be our bug, but it has pre-releases: 
![image](https://github.com/user-attachments/assets/312e7077-098f-4e12-90c8-bcaa503dca19)

sffamily replied:
will fix that.

sffamily commented:
@ntrogh I've updated and addressed all round 2 comments. pls help review.

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+    In the AI Toolkit view, select **TOOLS** > **Bulk Run** to open the Bulk Run view.
 
-## Run any prompt
+1. Select dataset
```
```suggestion
1. Select either a sample dataset or import a local JSONL file that has a `query` field to use as prompts.
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+1. Select dataset
 
-Once dataset is loaded in the bulk run view, you can find the table format of the imported dataset, with column of query and column of response.
+    You can select either a sample dataset to get familiar about the feature, or import a local JSONL file that has at least a field `query` to use as prompts.
```
```suggestion
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+    ![Select dataset](./images/bulkrun/dataset.png)
 
-Like in the playground, you can select the AI model, context for your prompt, and inference parameters.
+1. Run a single prompt
```
```suggestion
1. Once the dataset is loaded, select **Run** or **Rerun** on any prompt to run a single prompt.
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+1. Run a single prompt
 
-Initially the response cell of each row is empty with a `Run` icon. You can select this icon to execute prompt just for that row.
+    Once dataset is loaded in the bulk run view, you can **Run** or **Rerun** any prompt.
```
```suggestion
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+    ![Bulk run prompts](./images/bulkrun/bulkrun_one.png)
 
-## Rerun
+1. Bulk run prompts
```
```suggestion
1. Select **Run all** on the top of the Bulk Run view to automatically run through queries. The responses are shown in the **response** column.
```

ntrogh replied:
Also add a screenshot of the top of the screen and highlight the Run all button for clarity.

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+1. Bulk run prompts
 
-For any previously response, there is `Rerun` icon displayed for that row. You can switch to different model with different parameter to quickly see the results.
+    Select **Run all** on the top of bulk run view to automatically run through all queries and display responses in the response column. There is an option to only run the remaining queries that have not been executed.
```
```suggestion
    There is an option to only run the remaining queries that have not yet been run.
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
+    Select **Run all** on the top of bulk run view to automatically run through all queries and display responses in the response column. There is an option to only run the remaining queries that have not been executed.
 
-## Export results
+1. Export results
```
```suggestion
1. Select the **Export** button to export the results to a JSONL format.
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
-
-You can also import another dataset in JSONL format for the bulk run.
\ No newline at end of file
+    You can export the bulk run result to a JSONL format for future reference.
```
```suggestion

```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
-You can also import another dataset in JSONL format for the bulk run.
\ No newline at end of file
+    You can export the bulk run result to a JSONL format for future reference.
+    You can also import another dataset in JSONL format for the bulk run.
```
```suggestion
1. Select **Import** to import another dataset in JSONL format for the bulk run.
```

ntrogh commented on docs/intelligentapps/bulkrun.md:
```diff
@@ -9,34 +9,34 @@ MetaDescription: Run a set of prompts in an imported dataset, individually or in
 
-Select `Run all` on the top of bulk run view to automatically run through all queries and display responses in the response column.
-There is an option to only run the remaining queries that have not been executed.
+    ![Bulk run prompts](./images/bulkrun/bulkrun_one.png)
```
Highlight the button in the screenshot with a red rectangle to make it obvious for users.

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
 
 # Evaluation Overview
```
```suggestion
# Model evaluation
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
 ## Start an evaluation job
 
-You will be promoted to confirm you want to start the evaluation. Or select any individual evaluation job to re-run it.
+1. In AI Toolkit view, select **TOOLS** > **Evaluation** to open the evluation view.
```
```suggestion
1. In AI Toolkit view, select **TOOLS** > **Evaluation** to open the Evaluation view.
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
 
-You will be promoted to confirm you want to start the evaluation. Or select any individual evaluation job to re-run it.
+1. In AI Toolkit view, select **TOOLS** > **Evaluation** to open the evluation view.
+1. Select the button **Create Evaluation**, then answer the questions:
```
```suggestion
1. Select the **Create Evaluation** button and provide the following information:
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
+
+    - **Evaluation job name:** default or a name you can specify
+    - **Evaluator:** currently the built-in evaluators can be selected.
+    ![Evaluators](./images/evaluation/evaluators.png)
```
```suggestion
        ![Evaluators](./images/evaluation/evaluators.png)
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
+    - **Judging model:** a model from the list that can be selected as judging model to evaluate for some evaluators.
+    - **Dataset:** you can start with a sample dataset for learning purpose, or import a JSONL file with fields `query`,`response`,`ground truth`.
+1. Once you provide all necessary information for evaluation, a new evaluation job is created. You will be promoted to open your new evaluation job details.
+![Open evaluation](./images/evaluation/openevaluation.png)
```
```suggestion

    ![Open evaluation](./images/evaluation/openevaluation.png)
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
+![Open evaluation](./images/evaluation/openevaluation.png)
+
+1. Verify your dataset and select **Run Evaluation** to start the evaluation.
+![Run Evaluation](./images/evaluation/runevaluation.png)
```
```suggestion

    ![Run Evaluation](./images/evaluation/runevaluation.png)
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
 
-Once an evaluation job is started, you can find its status from the evaluation job overview. Each evaluation job has link to the dataset used, logs from evaluation process, timestamp and details of the evaluation.
+Once an evaluation job is started, you can find its status from the evaluation job view.
+![Running evaluation](./images/evaluation/running.png)
```
```suggestion

![Running evaluation](./images/evaluation/running.png)

```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
-Once an evaluation job is started, you can find its status from the evaluation job overview. Each evaluation job has link to the dataset used, logs from evaluation process, timestamp and details of the evaluation.
+Once an evaluation job is started, you can find its status from the evaluation job view.
+![Running evaluation](./images/evaluation/running.png)
+Each evaluation job has link to the dataset used, logs from evaluation process, timestamp and link to the details of the evaluation.
```
```suggestion
Each evaluation job has a link to the dataset that was used, logs from the evaluation process, timestamp, and a link to the details of the evaluation.
```

ntrogh commented on docs/intelligentapps/evaluation.md:
```diff
@@ -11,26 +11,38 @@ MetaSocialImage:
 
-You can also open the data in `Data Wrangler` extension from the button `Open in Data Wrangler`
\ No newline at end of file
+You can also select **Open In Data Wrangler** to open the data in `Data Wrangler` extension.
```
```suggestion
You can also select **Open In Data Wrangler** to open the data with the Data Wrangler extension.
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -13,53 +13,60 @@ MetaSocialImage:
 
-You can filter the models in the model catalog by using the following criteria:
-- Hosted by:
+1. use the filters to reuce the list of available models
```
```suggestion
1. Use the filters to reduce the list of available models
```

ntrogh commented on docs/intelligentapps/models.md:
```diff
@@ -13,53 +13,60 @@ MetaSocialImage:
-You can filter the models in the model catalog by using the following criteria:
-- Hosted by:
+1. use the filters to reuce the list of available models
+    ![Select model in model catalog](./images/models/model_catalog.png)
```
Put the screenshot after the list of filters

ntrogh reviewed (changes requested):
@sffamily Thanks for the updates. Just a couple small updates remaining.

sffamily commented:
Thank you @ntrogh for all the comments, I've updated according to the feedback.

ntrogh reviewed (approved):
@sffamily Looking good. Thanks a lot for all the updates!
